src/app/fdctl/run/tiles/fd_shred.c

```diff
@@ -124,6 +124,13 @@ typedef struct {
   fd_fec_resolver_t  * resolver;
   fd_pubkey_t          identity_key[1]; /* Just the public key */
 
+  ulong                round_robin_id;
+  ulong                round_robin_cnt;
+  /* Number of batches we shredded from PoH during the current slot */
```

Comment:
"we shredded" seems to suggest maybe just this tile, but this is for any tile, right?